tests/compatibility-tests/pom.xml

```diff
@@ -265,6 +271,39 @@
       </plugins>
    </build>
    <profiles>
+      <profile>
+         <id>compatibility-real-servers</id>
```

Comment:
What about _compatibility-tests-distribution_ so that it shares a nicer prefix with the other profile, and is more descriptive of the difference between them?

Also, with this having its own profile, its likely not going to actually run in any CI jobs currently, that intended?

Reply:
@gemmellr I certainly don't intend it to run on GitHub CI.

I plan to change our internal CI at Red Hat to use this profile.